.example-env

```diff
@@ -0,0 +1,4 @@
+PG_Username =
+PG_Password =
+PG_Source = 
+API_KEY =
```

Comment:
All files should end with a new line, that's why that red sign is in here. Try to add that to the eslint config.